### PR TITLE
Enable service 1788

### PR DESCRIPTION
**Pull request by elliotcm**
image

https://trello.com/c/HBvlmYak/1022-adding-new-service-to-local-links-manager

### CODE REVIEW

File: db/migrate/20190603131841_enable_service1788.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class EnableService1788 < ActiveRecord::Migration[5.2]
+  def change
```
Comment (thomasleese) [inline db/migrate/20190603131841_enable_service1788.rb]: Should this be an `up`?

File: docs/adding-a-new-lgsl-code.md [hunk 2]
```diff
@@ -18,4 +26,20 @@ Providing tiers can be:
 - `county/unitary`
 - `district/unitary`
 
-Once your pull request has been merged the codes will become active overnight.
+## Activate relevant service interactions
+You can either do this by creating a local transaction in Publisher, but this
+may show the transaction on the live site before the links are ready, so check
+with your product manager to make sure this is OK, or use a migration instead.
+
+See db/migrate/20190603131841_enable_service1788.rb for an example of the data
```
Comment (thomasleese) [inline docs/adding-a-new-lgsl-code.md]: It would be cool if this could be a link to the file.

File: docs/adding-a-new-lgsl-code.md [hunk 3]
```diff
@@ -18,4 +26,20 @@ Providing tiers can be:
 - `county/unitary`
 - `district/unitary`
 
-Once your pull request has been merged the codes will become active overnight.
+## Activate relevant service interactions
```
Comment (thomasleese) [inline docs/adding-a-new-lgsl-code.md]: You started with `1.`, `2.` in your headings above but they seem to be missing from here onwards, should they be in these ones too?

File: db/migrate/20190603131841_enable_service1788.rb (hunk 1 above)
Comment (elliotcm) [reply db/migrate/20190603131841_enable_service1788.rb]: Updated.

File: docs/adding-a-new-lgsl-code.md (hunk 2 above)
Comment (elliotcm) [reply docs/adding-a-new-lgsl-code.md]: Apparently it can.

File: docs/adding-a-new-lgsl-code.md (hunk 3 above)
Comment (elliotcm) [reply docs/adding-a-new-lgsl-code.md]: Updated.

**Review by thomasleese — APPROVED** [review]

**Pull request merged**